kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -11,6 +11,8 @@ html, body {
 #wrapper {
     min-height: 100%;
     position: relative;
+    display: block;
+    overflow: auto;
```

Comment:
Could we change this to overflow: visible? The auto behaviour gives some ugly scrollbars that go kinda screwy when the tooltip disappears again.

With visible rather than auto, it looks like this:
![image](https://cloud.githubusercontent.com/assets/1680573/6406393/3898d1da-be56-11e4-8754-e7982c54c694.png)